src/main/java/seedu/duke/command/ListEventCommand.java

```diff
@@ -25,10 +24,12 @@
     public static final String COMMAND_USAGE = COMMAND_WORD + ": List all the events in the Timetable. Parameters: "
             + "[" + PREFIX_DELIMITER + PREFIX_TIMING + " YYYY or YYYY-MM]";
 
-    private static final String COMMAND_SUCCESSFUL_MESSAGE = "These are the events in the specified time period: "
-            + Formatter.LS;
-    private static final String COMMAND_UNSUCCESSFUL_MESSAGE = "Failed to find any events in the specified time period."
-            + Formatter.LS;
+    private static final String COMMAND_SUCCESSFUL_GENERIC_MESSAGE = "These are the events stored: ";
+    private static final String COMMAND_UNSUCCESSFUL_GENERIC_MESSAGE = "There are no events stored in the timetable!";
+    private static final String COMMAND_SUCCESSFUL_TIME_PERIOD_MESSAGE = "These are the events "
+            + "in the specified time period: ";
+    private static final String COMMAND_UNSUCCESSFUL_TIME_PERIOD_MESSAGE = "Failed to find any events "
+            + "in the specified time period.";
```

Comment:
Maybe can add the time period parameter?

Reply:
He did that in the execute return, since this is a private static final, the time period cannot be added here.